plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTableProperties.java

```diff
@@ -173,7 +177,16 @@ public HiveTableProperties(
                         PARTITION_PROJECTION_LOCATION_TEMPLATE,
                         "Partition projection location template",
                         null,
-                        false));
+                        false),
+                new PropertyMetadata<>(
+                        EXTRA_PROPERTIES,
+                        "Extra table properties",
```

Comment:
I'm not sure that I agree with "arbitrary", because these properties presumably have meaning- just not one that Trino understands. Maybe... "additional_properties"? It's pretty close to "extra_properties", but maybe slightly clearer than "extra". Naming is hard.

Reply:
Based on offliine discussion we finalized the property name to `extra_properties`